lms/djangoapps/grades/tasks.py

```diff
@@ -0,0 +1,54 @@
+"""
+This module contains tasks for asynchronous execution of grade updates.
+"""
+
+from celery import task
+from django.contrib.auth.models import User
+
+from lms.djangoapps.course_blocks.api import get_course_blocks
+from lms.djangoapps.courseware.courses import get_course_by_id
+from opaque_keys.edx.keys import UsageKey
+from opaque_keys.edx.locator import CourseLocator
+from openedx.core.djangoapps.content.block_structure.api import get_course_in_cache
+
+from .config.models import PersistentGradesEnabledFlag
+from .transformer import GradesTransformer
+from .new.subsection_grade import SubsectionGradeFactory
+
+
+@task()
+def recalculate_subsection_grade(**kwargs):  # pylint: disable=unused-argument
+    """
+    Updates a saved subsection grade.
+    This method expects that the kwargs dictionary will contain the following
+    entries:
+       - user: serialized id of applicable User object
+       - course_id: Unicode string representing the course
+       - usage_id: Unicode string indicating the courseware instance
```

Comment:
Since the args are always the same and all required, they could probably be laid out explicitly. The `SCORE_CHANGED` signal does contain extra information (`points_possible` and `points_earned`) that you don't need here.